Tensile/BenchmarkProblems.py

```diff
@@ -236,58 +242,103 @@ def benchmarkProblemType(problemTypeConfig, problemSizeGroupConfig, problemSizeG
 
         pushWorkingPath(shortName)
         stepBaseDir = globalParameters["WorkingPath"]
-        pushWorkingPath("source")
-
-        # enumerate benchmark permutations and create resulting solution objects
-        forkPermutations = constructForkPermutations(benchmarkStep.forkParams)
-        maxPossibleSolutions = len(forkPermutations)
-
-        regSolutions = generateForkedSolutions(benchmarkProcess.problemType, \
-                benchmarkStep.constantParams, forkPermutations)
-        kcSolutions = generateCustomKernelSolutions(benchmarkProcess.problemType, \
-                benchmarkStep.customKernels, not benchmarkStep.customKernelWildcard)
 
-        maxPossibleSolutions += len(kcSolutions)
-        solutions = regSolutions + kcSolutions
+        # file paths
+        resultsFileBase = os.path.normpath(os.path.join( \
+                globalParameters["WorkingPath"], "../Data", shortName))
+        if benchmarkStep.isFinal():
+            resultsFileBaseFinal = resultsFileBase
+        resultsFileName = resultsFileBase + ".csv"
+        solutionsFileName = resultsFileBase + ".yaml"
 
-        print1("# Actual Solutions: {} / {} after SolutionStructs\n" \
-            .format(len(solutions), maxPossibleSolutions))
+        # check if we can and should use cache file
+        cachePath = os.path.join(stepBaseDir, "cache.yaml")
+        pushWorkingPath("source")
 
-        # handle no valid solutions
-        if len(solutions) == 0:
-            msg = "Your parameters resulted in 0 valid solutions."
-            if globalParameters["PrintSolutionRejectionReason"]:
-                msg += "\nExamine reject and backtrace messages above to see why" \
-                        "and where solutions were rejected."
+        cache = False
```

Comment:
A future user may not immediately catch why we need this cache param here. A short comment would be great.

Reply:
I updated the comment a few lines above this and changed the variable name to hopefully be more clear